frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -283,7 +311,7 @@ const PRInputs = ({
   return (
     <div className={styles.inline}>
       <label className={styles.bold}>
-        {label} {!isTpm && <span className={styles.red_color}>*</span>}
+        {label} {!isTpm && !openOther && <span className={styles.red_color}>*</span>}
```

Comment:
I really like that you're thinking about the UX here and hiding the asterisk when the user opens up the other PR's field 👍.